xml/System.Diagnostics.Tracing/EventSource.xml

```diff
@@ -295,8 +304,11 @@
 ## Remarks  
  Traits provide additional information for an event and are interpreted by an event listener. Traits are specified as key-value pairs and if not specified in pairs, an exception will occur. You can retrieve a trait by passing the key to the <xref:System.Diagnostics.Tracing.EventSource.GetTrait%2A>, method.  
   
- Specifying `settings` when the <xref:System.Diagnostics.Tracing.EventSource> is constructed enables you to  specify whether the event is written in a manifest-based or a self-describing format. In addition, you can specify that an exception should be raised when an error occurs during the event-writing process.  
-  
+ Specifying `settings` when the <xref:System.Diagnostics.Tracing.EventSource> is constructed enables you to specify whether the event is written in a manifest-based or a self-describing format. In addition, you can specify that an exception should be raised when an error occurs during the event-writing process.
+
+ When using this constructor, ensure that the `eventSourceName` argument matches the ETW name defined by the <xref:System.Diagnostics.Tracing.EventSourceAttribute> attribute on that type. Otherwise, the GUIDs returned by the <xref:System.Diagnostics.Tracing.EventSource.Guid> property and the <xref:System.Diagnostics.Tracing.EventSource.GetGuid(System.Type)> method will be different.
```

Comment:
This constructor doesn't have an `eventSourceName` parameter, so this will have to be reworded a bit.